ivy/functional/ivy/gradients.py

```diff
@@ -812,6 +812,17 @@ def lars_update(
     -------
     ret
         The new function weights ws_new, following the LARS updates.
+     Examples
+        --------
+        With :code:`ivy.Array` inputs:
+        >>> w = ivy.array([[[5., 3., 2.], [0., 4., 1.], [-2., 3., -1.]]])
+        >>> dcdw = ivy.array([[[0.5, 0.92, 0.1], [0.2, 0.7, 0.3], [0.3, 0.8, 0.01]]])
+        >>> lr = ivy.array(0.3)
+        >>> ws_new = w.lars_update(dcdw, lr, inplace=False)
```

Comment:
This is using the array instance method here we should use the functional implementation as outline above